nemo/collections/asr/modules/activations.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import torch
+import torch.nn as nn
+
+__all__ = ['Swish']
+
+
+class Swish(nn.Module):
+    """
+    Swish activation function introduced in 'https://arxiv.org/abs/1710.05941'
+    """
+
+    def forward(self, x):
```

Comment:
But more importantly, @VahidooX I was thinking, should we switch to the more memory efficient implementation from here (with proper attribution) - https://twitter.com/jeremyphoward/status/1188304165971218432?s=20

I can look into writing the jitted fused op for this if you agree the memory and execution time savings are worth it.

Reply:
I can take the timing and memory consumption to see which one is better. What is a good profiler to compare such stuff?

Reply:
Pytorch 1.6 has a memory profiler + time profiler I think. But its not necessary for this PR. Lets add it in another PR.